### PR TITLE
upower: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Sorry, but the cla needs my private email. Who will be able to see that and where will it be listed (eg a contributors file somewhere)

File: pages/linux/upower.md
Comment (bl-ue) [inline pages/linux/upower.md:3]:
```suggestion
> System utility to provide battery information and statistics.
```

**Comment by CleanMachine1** [comment]
I am perfectly happy to sign the license agreement however I am not willing to give my private email to it

File: pages/linux/upower.md
Comment (bl-ue) [inline pages/linux/upower.md:22]:
```suggestion
- Display version:
```

**Comment by bl-ue** [comment]
>Sorry, but the cla needs my private email. Who will be able to see that and where will it be listed (eg a contributors file somewhere)

@CleanMachine1 None of us have access to your email.  I'm not sure what cla-assistant.io does with it, but they certainly keep it private (I don't think any human has access to it on any side).

**Comment by CleanMachine1** [comment]
Is it mandatory for it to be done?

**Comment by bl-ue** [comment]
Yes, it is, for legal reasons 😞

There's absoluately **no** danger of your email being exposed though!

**Comment by marchersimon** [comment]
@CleanMachine1 you could however use another email for your commits and add it to your GitHub account.

**Comment by CleanMachine1** [comment]
Don't close this PR and I will have it signed or whatever in the next 24 hours (probably)

**Review by nicokosi — APPROVED** [review]
:+1: Looks good to me!

**Comment by CleanMachine1** [comment]
Read up and see! Good to here its alright though

**Review by bl-ue — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Do you know if I can remove CLA assistant from my account after this is merged?

**Comment by bl-ue** [comment]
Do you mean unsign the CLA?

**Comment by bl-ue** [comment]
Ah yes, I'm sure you can do that.  But you'll need to do it all over again if you want to open another PR... 😄

**Comment by CleanMachine1** [comment]
Ok, well considering it took me 20 seconds to think of topic to cover, I will find another command I use which isn't here.

Overall been enjoying this repo, been using it for around a week.
I have a suggestion I will add, it is for gpg

I often encrypt single files and the command is gpg -c file

File: pages/linux/upower.md
Comment (marchersimon) [inline pages/linux/upower.md:4]:
```suggestion
> More information: <https://upower.freedesktop.org/docs/upower.1.html>.
```
I think we should directly point to the man page. Finding the home page from there is easier than the other way around.

File: pages/linux/upower.md
Comment (marchersimon) [inline pages/linux/upower.md:10]: I'm not really happy with this description. When I run that command, I get:
```
.../line_power_AC0
.../battery_BAT0
.../DisplayDevice
```
I wouldn't call `line power` and `battery` battery-using. However, there isn't really good documentation for this and the only description I found was `enumerate all power objects on the system`. 
How about:
```suggestion
- List all power objects on the system:
```

File: pages/linux/upower.md
Comment (marchersimon) [inline pages/linux/upower.md:18]: Since both examples are almost the same I would also phrase them the same.
```suggestion
- Watch for and print battery status changes:

`upower --monitor`

- Watch for and print detailed battery status changes:
```

**Review by marchersimon — CHANGES_REQUESTED** [review]
The whole command isn't just about the battery, but about power in general (including line power, the display, etc.), so I would replace `battery` with `power` everywhere.

File: pages/linux/upower.md
Comment (marchersimon) [reply pages/linux/upower.md:10]: That's fine too.

**Review by CleanMachine1 — COMMENTED** [review]
All fixed and changed. You can submit review in my opinion :)

File: pages/linux/upower.md
Comment (marchersimon) [inline pages/linux/upower.md:3]: I think we should change this too. Either:
```suggestion
> System utility to provide power and battery information and statistics.
```
or 
```suggestion
> System utility to provide power information and statistics.
```

Comment (CleanMachine1) [reply pages/linux/upower.md]: Commited option 1

**Comment by marchersimon** [comment]
We usually wait a day or two to give anyone a chance to comment.

**Comment by CleanMachine1** [comment]
Makes sense, I understand.

I have a question unrelated to upower, rather just the repo.

I see a lot of people lost collaborator status today, how hard is it to gain it since I am very interested in this repo, ever since starting Linux, I always found it an annoyance to scroll through man pages.

I may start to commit and help more, if you would like me to.

Thanks

**Comment by CleanMachine1** [comment]
Can collab leave reviews, like what you have done. I like the sound of playing a role in helping!

**Comment by bl-ue** [comment]
Collaborator or organization member.  All you need is write access.

Btw, I'm preparing a reply for you, so be patient ;P

**Comment by bl-ue** [comment]
@CleanMachine1

>I see a lot of people lost collaborator status today

That's not true, exactly.  @marchersimon opened issues to discuss whether or not we should remove them, but they're

They were interested in the repo at various times over the years, so they were invited as collaborators.  As with everyone though, their interests shifted onto other things, so they became less and less active.  It's been a while, so they aren't helping out with the maintenance tasks nor contributing pages, so @marchersimon wants to remove them, **not as a demotion or punishment** or anything of the sort, but **as a reflection of their actual involvement in the project** :)

>how hard is it to gain

Not hard at all! Usually, when we see someone contributor several pages (5 is the minimum per [`COMMUNITY-ROLES.md`](https://github.com/tldr-pages/tldr/blob/b575ffe207039a4fafa0d6e406a182b9c0316505/COMMUNITY-ROLES.md#when-to-change-roles)), and that person seems particularly active, we open an issue inviting them.

>I am very interested in this repo, ever since starting Linux, I always found it an annoyance to scroll through man pages.

❤️

>I may start to commit and help more, if you would like me to.

That would be fantastic!  Contribution of any kind from any contributor is desired, as this is a community volunteer-based project (i.e. the maintainers were and are just `tldr` users).

>Can collab leave reviews, like what you have done. I like the sound of playing a role in helping!

Any user with a GitHub account can leave reviews on any PR in any public repository.  They can comment, suggested code changes, etc.  Really, you don't even need to be a collaborator to help out with tldr.  But collaborators and organization owners (both roles have write access to the repo) can label and close issues/PRs, and merge PRs, in addition to the common things that every user can do.

>I like the sound of playing a role in helping!

😍 That would be amazing!  Please, open a few more PRs with some nice, high-quality pages (and maybe help out fix a few existing open PRs) and you'll be elligible in no time.

**Comment by CleanMachine1** [comment]
Well the reason I am so interested is I (for some reason) like to make documentation.
I always hate products that have terrible documentation, software is no exception.
I will probably produce some really small branch, I am currently just about to push a branch which changes a certain word.

I don't expect to gain collab or anything similar from these commits however it would be nice :smile: 

Expect me to keep following this, even if I stop for a week, continue to ping me with review requests as I will happily do them ASAP.

File: pages/linux/upower.md
Comment (CleanMachine1) [reply pages/linux/upower.md:6]: Good change!

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Push?

File: pages/linux/upower.md
Comment (marchersimon) [reply pages/linux/upower.md:4]: We only use `manned.org` if there isn't an official source.

**Review by marchersimon — APPROVED** [review]

File: pages/linux/upower.md
Comment (bl-ue) [reply pages/linux/upower.md:4]: Actually, manned.org looks better, and since they have exactly the same content, I think we should use manned.org.  wdyt?

Comment (CleanMachine1) [reply pages/linux/upower.md:4]: The official page isn't just a man page, it is more

Comment (bl-ue) [reply pages/linux/upower.md:4]: Then let's use the official site 👍🏻

**Review by bl-ue — APPROVED** [review]

**Review by Waples — APPROVED** [review]

**Comment by Waples** [comment]
No need to be so impatient @CleanMachine1.

**Comment by navarroaxel** [comment]
Hey @CleanMachine1 join us in our https://gitter.im/tldr-pages/tldr channel

**Pull request merged**
